ailab/models/openai.py

```diff
@@ -1,59 +1,61 @@
 """"Fetch embeddings from the Microsoft Azure OpenAI API"""
+
 import os
-import openai
+
 import tiktoken
+from dotenv import load_dotenv
+from openai import AzureOpenAI
 
-import dotenv
-dotenv.load_dotenv()
+load_dotenv()
 
 # https://learn.microsoft.com/en-us/azure/cognitive-services/openai/how-to/embeddings?tabs=python
 
+
 def safe_get(key):
     value = os.environ.get(key)
     if not value:
         raise Exception(f"Environment variable {key} not defined")
```

Comment:
I suggest the use of `KeyError` or a custom exception instead of `Exception` to be more precise on the nature of the exception.

Reply:
Agreed. I would suggest a round of refactor. #92